### PR TITLE
analytic content for oracle weblogic

### PR DESCRIPTION
**Pull request by spsocprime**
## Proposed Changes
Analytic content for Oracle Weblogic Server

### CODE REVIEW

**Comment by v-sabiraj** [comment]
Please update 'OracleWebLogicServerEvent' to ".script/tests/detectionTemplateSchemaValidation/ValidConnectorIds.json"

**Comment by NikTripathi** [comment]
@aprakash13 Please review Analytic rules.

**Comment by v-sabiraj** [comment]
Please update 'OracleWebLogicServerEvent' to ["ValidConnectorIds.json"](https://github.com/Azure/Azure-Sentinel/blob/master/.script/tests/detectionTemplateSchemaValidation/ValidConnectorIds.json)

**Comment by v-sabiraj** [comment]
Please add version and kind for Analytic Rules of **"Oracle - Multiple client errors from single IP"** and **"Oracle - Multiple server errors from single IP"**, thanks.

**Comment by vmanojreddy** [comment]
@aprakash13  Please review Analytic rules.

File: Solutions/OracleWebLogicServer/Analytic Rules/OracleWebLogicMultipleClientErrorsFromSingleIP.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+id: 41775080-5740-11ec-bf63-0242ac130002
+name: Oracle - Multiple client errors from single IP
+description: |
+  'Detects multiple client errors from one source in short timeframe'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: OracleWebLogicServer
+    dataTypes:
+      - OracleWebLogicServerEvent
```
Comment (vmanojreddy) [inline Solutions/OracleWebLogicServer/Analytic Rules/OracleWebLogicMultipleClientErrorsFromSingleIP.yaml:9]: Please add queryFrequency, queryPeriod, triggerOperator and triggerThreshold.

File: Solutions/OracleWebLogicServer/Analytic Rules/OracleWebLogicMultipleServerErrorsRequestsFromSingleIP.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+id: 268f4fde-5740-11ec-bf63-0242ac130002
+name: Oracle - Multiple server errors from single IP
+description: |
+  'Detects multiple server errors from one source in short timeframe'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: OracleWebLogicServer
+    dataTypes:
+      - OracleWebLogicServerEvent
```
Comment (vmanojreddy) [inline Solutions/OracleWebLogicServer/Analytic Rules/OracleWebLogicMultipleServerErrorsRequestsFromSingleIP.yaml:9]: Please add queryFrequency,queryPeriod,triggerOperator and triggerThreshold

**Review by vmanojreddy — CHANGES_REQUESTED** [review]

**Review by aprakash13 — CHANGES_REQUESTED** [review]
Queries look great. A few small suggestions provided.

Also, please get the validation errors fixed before we can merge this. Thanks.

File: Solutions/OracleWebLogicServer/Analytic Rules/OracleWebLogicMultipleClientErrorsFromSingleIP.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+  - T1133
+query: |
+  let threashold = 100;
```
[the remark this answers is not in the document]
Comment (spsocprime) [reply Solutions/OracleWebLogicServer/Analytic Rules/OracleWebLogicMultipleClientErrorsFromSingleIP.yaml:20]: fixed typo

File: Solutions/OracleWebLogicServer/Analytic Rules/OracleWebLogicMultipleServerErrorsRequestsFromSingleIP.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+triggerThreshold: 0
+tactics:
+  - Impact
+  - InitialAccess
+relevantTechniques:
+  - T1498
+  - T1190
+  - T1133
+query: |
+  let threashold = 100;
```
[the remark this answers is not in the document]
Comment (spsocprime) [reply Solutions/OracleWebLogicServer/Analytic Rules/OracleWebLogicMultipleServerErrorsRequestsFromSingleIP.yaml:22]: fixed typo

File: Solutions/OracleWebLogicServer/Hunting Queries/OracleWebLogicCriticalEventSeverity.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+  - T1133
+query: |
+  OracleWebLogicServerEvent
+  | where TimeGenerated > ago(24h)
+  | where isnotempty(EventSeverity)
+  | where EventSeverity contains "CRITICAL" or tostring(EventSeverity) contains 'ALERT' or tostring(EventSeverity) contains 'EMERGENCY'
```
[the remark this answers is not in the document]
Comment (spsocprime) [reply Solutions/OracleWebLogicServer/Hunting Queries/OracleWebLogicCriticalEventSeverity.yaml:19]: fixed

File: Solutions/OracleWebLogicServer/Hunting Queries/OracleWebLogicErrors.yaml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  - connectorId: OracleWebLogicServer
+    dataTypes:
+      - OracleWebLogicServerEvent
+tactics:
+  - Defense Evasion
+relevantTechniques:
+  - T1070
+query: |
+  OracleWebLogicServerEvent
+  | where EventMessage contains "error" or EventMessage contains "Invalid"
```
[the remark this answers is not in the document]
Comment (spsocprime) [reply Solutions/OracleWebLogicServer/Hunting Queries/OracleWebLogicErrors.yaml:16]: fixed

File: Solutions/OracleWebLogicServer/Analytic Rules/OracleWebLogicMultipleServerErrorsRequestsFromSingleIP.yaml (hunk 2 above)
Comment (spsocprime) [reply Solutions/OracleWebLogicServer/Analytic Rules/OracleWebLogicMultipleServerErrorsRequestsFromSingleIP.yaml:9]: fixed

**Pull request merged**